diagnostic_msgs/QUALITY_DECLARATION.md

```diff
@@ -88,11 +88,11 @@ Results of the nightly linter tests can be found [here](http://build.ros2.org/vi
 
 ### Direct Runtime ROS Dependencies [5.i]/[5.ii]
 
-`diagnostic_msgs` has the following runtime ROS dependencies:
-* `builtin_interfaces`
-* `geometry_msgs`
-* `rosidl_default_runtime`
-* `std_msgs`
+`diagnostic_msgs` has the following runtime ROS dependencies, which are at or above Quality Level 3:
+* `builtin_interfaces`: [QL 3](https://raw.githubusercontent.com/ros2/rcl_interfaces/master/builtin_interfaces/QUALITY_DECLARATION.md)
```

Comment:
Were these raw.githubusercontent.com added on purpose?

Reply:
Switched to github.com